src/m_lib/setup.py

```diff
@@ -10,4 +11,5 @@
 setup(
   name = "pictorial",
   cmdclass = {"build_ext": build_ext},
-  ext_modules = ext_modules)
\ No newline at end of file
+  include_dirs = [numpy.get_include()]
```

Comment:
A comma is needed at the end of line so final line should look like this
`include_dirs = [numpy.get_include()],`